WebCLI/forms.py

```diff
@@ -83,12 +83,6 @@ def clean_structure(self):
             self.add_error('structure', 'Atom syntax (one atom per line): Li 0.0 0.0 1.6')
         return structure
 
-    def clean_active_orbitals(self):
-        active_orbitals = self.clean().get('active_orbitals')
-        if not qm.molecule.validate_orbitals_syntax(active_orbitals):
-            self.add_error('active_orbitals', 'Orbital syntax (one orbital per line): A1 1 2 4 5 7')
-        return active_orbitals
-
```

Comment:
Shouldn't active orbitals still be validated in case user does not leave it blank?